src/LandPlanner.cpp

```diff
@@ -24,20 +24,28 @@ LandPlanner::LandPlanner(
       transform_wrapper_(),
       state_(LandState::DONE),
       requested_height_(0.0),
+      cushion_height_(0.0),
+      actual_height_(0.0),
       actual_descend_rate_(0.0),
       descend_rate_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
               "descend_rate")),
+      cushion_rate_(ros_utils::ParamUtils::getParam<double>(
+              private_nh,
+              "cushion_rate")),
       descend_acceleration_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
               "descend_acceleration")),
+      cushion_acceleration_(ros_utils::ParamUtils::getParam<double>(
+              private_nh,
+              "cushion_acceleration")),
       last_update_time_(),
       startup_timeout_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
               "startup_timeout")),
       update_timeout_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
-              "update_timeout")),
+              "update_timeout")),                    
       uav_arm_client_(nh.serviceClient<iarc7_msgs::Arm>("uav_arm"))
 {
     landing_detected_subscriber_ = nh.subscribe("landing_detected",
```

Comment:
Can a ROS_ASSERT be added here to check that cushion_acceleration, cushion_rate_, descend_rate_, and descend_acceleration have the appropriate sign? I'd hate for a crash to occur because they were wrong. Cushion_rate and cushion_accel are particularly susceptible to being confused IMO.